### PR TITLE
 u/index.md

### PR DESCRIPTION
**Pull request by Ainatiana**
Fix #39 
Hello teammate!
we have done this new PR.
Please, have a look at it.
Daniatiana

### CODE REVIEW

**Review by ndorfin — CHANGES_REQUESTED** [review]
Much better. Please add a link back to the root index, that Sugi and Clopedia are adding.

File: u/index.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # List of artists starting with U:
 
-  Oops! It seems as if there aren't any Malagasy artists with names starting with 'U'. 
+
+Oops! It seems as if there aren't any Malagasy artists with names starting with 'U'.
+
+[Go back to the artist index](https://github.com/onja-madagascar/music-of-madagascar/blob/a8f5944d0cbbde6b1d34cbe3783974d753f4bc10/index.md)
```
Comment (ndorfin) [inline u/index.md]: This needs to be a relative URL

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — APPROVED** [review]
Looking good. 🚢-it!

**Comment by Ainatiana** [comment]
Thank you so much!

**Review by ndorfin — APPROVED** [review]
Looks good, 🚢 -it!

**Pull request merged**